content/engine/security/trust/trust_key_mng.md

```diff
@@ -29,10 +29,9 @@ Delegation keys are optional, and not generated as part of the normal `docker`
 workflow.  They need to be
 [manually generated and added to the repository](trust_delegation.md#creating-delegation-keys).
 
->**Note**:
+>**Note**
 >
-> Prior to Docker Engine 1.11, the snapshot key was also generated and stored locally client-side.
-> Use the Notary CLI to [manage your snapshot key locally again](https://github.com/theupdateframework/notary/blob/master/docs/advanced_usage.md#rotate-keys) for repositories created with newer versions of Docker.
+> Use the Notary CLI to [manage your snapshot key locally again](https://github.com/theupdateframework/notary/blob/master/docs/advanced_usage.md#rotate-keys)..
```

Comment:
Double period here. And the sentence here is lonely now that the version callout was removed. Maybe we should say something like "You can rotate snapshot keys locally using the Notary CLI, see..."

probably wouldn't make it a note in that case though. It could go together with the bullet on line 24 maybe

```suggestion
> Use the Notary CLI to [manage your snapshot key locally again](https://github.com/theupdateframework/notary/blob/master/docs/advanced_usage.md#rotate-keys).
```